app/views/fields/belongs_to/_show.html.erb

```diff
@@ -16,5 +16,5 @@ By default, the relationship is rendered as a link to the associated object.
 %>
 
 <% if field.data %>
-  <%= link_to field.data, [Administrate::NAMESPACE, field.data] %>
+  <%= link_to field.associated_resource, [Administrate::NAMESPACE, field.data] %>
```

Comment:
I like `displayed_...`, because it's a noun instead of a verb. The method is returning a String, instead of taking action to display the string itself.